### PR TITLE
Config updates for English fallbacks

### PR DESCRIPTION
**Pull request by balderdash**

### CODE REVIEW

File: pegasus/sites.v3/code.org/views/homepage_hero.haml [hunk 1]
```diff
@@ -60,7 +60,7 @@
       .col-66.desktop-feature{style: "position:relative;"}
         - heroes_arranged.each_with_index do |entry, index|
           - unless entry[:text].nil_or_empty? || show_census_layout
-            - text = I18n.t(entry[:text])
+            - text = "#{I18n.t(entry[:text])}"
```
Comment (Hamms) [inline pegasus/sites.v3/code.org/views/homepage_hero.haml]: why is this necessary?

Comment (balderdash) [reply pegasus/sites.v3/code.org/views/homepage_hero.haml]: The string gets modified a couple lines down, and it started throwing errors because it was trying to modify a frozen string. I still don't understand why it changed from not frozen to frozen, but it doesn't seem ideal to be modifying a localized string.

Looking at it again, I should probably have just used `.dup` instead of this string interpolation.

**Comment by Hamms** [comment]
[If time: Remove all untranslated English strings from other locales (these are just direct copies from the English files, and we can infer from there)](https://trello.com/c/UpQqX5vD/218-if-time-remove-all-untranslated-english-strings-from-other-locales-these-are-just-direct-copies-from-the-english-files-and-we-ca)
